symfony/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM skilldlabs/docker-phpcs-drupal:latest
+
+RUN composer global require escapestudios/symfony2-coding-standard:~2.0 \
+    && ln -s /root/.composer/vendor/escapestudios/symfony2-coding-standard/Symfony2 /root/.composer/vendor/squizlabs/php_codesniffer/CodeSniffer/Standards/Symfony2
```

Comment:
I suggest to override `CMD` to SF standard https://github.com/skilld-labs/docker-phpcs-drupal/blob/master/Dockerfile#L49

Reply:
Updated